canvas_sdk/v1/data/billing.py

```diff
@@ -0,0 +1,58 @@
+from typing import TYPE_CHECKING, Type
+
+from django.db import models
+
+from canvas_sdk.v1.data.base import TimeframeLookupQuerySet, ValueSetLookupQuerySet
+from canvas_sdk.v1.data.note import Note
+from canvas_sdk.v1.data.patient import Patient
+from canvas_sdk.value_set.value_set import CodeConstants
+
+if TYPE_CHECKING:
+    from canvas_sdk.value_set.value_set import ValueSet
+
+
+class BillingLineItemQuerySet(ValueSetLookupQuerySet, TimeframeLookupQuerySet):
+    """A class that adds functionality to filter BillingLineItem objects."""
+
+    def find(self, value_set: Type["ValueSet"]) -> models.QuerySet:
+        """
+        This method is overridden to use for BillingLineItem CPT codes.
+        The codes are saved as string values in the BillingLineItem.cpt field,
+        which differs from other coding models.
+        """
+        values_dict = value_set.values
+        return self.filter(cpt__in=values_dict.get(CodeConstants.HCPCS, []))
```

Comment:
Can you accomplish this override by overriding the `q_object` method rather than `find`? That's how I was able to point the Questionnaire query set class at a field on the model instead of codings. It's a bit more narrowly targeted than overriding `find`.

Reply:
I just looked into this, and for this specific case, I think overriding `find` is the better choice. This is because the `BillingLineItem` model isn't structured like any other codings tables. It has a single `cpt` field in the base table. Since we only want to allow one coding system for these lookups, I think it's better to do what I have than add additional logic to do that in a `q_object`.

Reply:
Yeah, there are a few point where this could be overridden, and it's debatable where right place will be for each use case. You can override `find`, `codings`, or `q_object`.

FWIW, the questionnaire scenario is almost the same as this one, where the fields are stored on the base table rather than in a separate codings table.